dbt/include/global_project/macros/materializations/archive/archive.sql

```diff
@@ -174,7 +174,8 @@
       identifier=target_table) -%}
 
   {%- if source_relation is none -%}
-    {{ exceptions.missing_relation(source_relation) }}
+    {%- set source_relation_name = [source_schema, source_table]|join(".") -%}
```

Comment:
In our Wilt Chamberlain release, we're going to make archive database-aware, meaning that this will become `database.schema.table`. I think this is ok for now, but wanted to raise that point for when we revisit this in the future. See here: https://github.com/fishtown-analytics/dbt/issues/838